src/cmd/build.js

```diff
@@ -56,8 +99,17 @@ async function defaultPackageCreator(
     filter: (...args) => fileFilter.wantFile(...args),
   });
 
+  let messageFile: string;
+  let extensionName: string = manifestData.name;
+
+  if (manifestData.default_locale) {
+    messageFile = path.join(sourceDir, '_locales',
```

Comment:
I think you can declare this variable here since it's not used outside of the if statement anymore. Like: `let messageFile = path.join...`